tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/ITestMethod.cs

```diff
@@ -5,13 +5,14 @@
 
 using System;
 using System.Reflection;
+using Datadog.Trace.DuckTyping;
 
 namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Testing.MsTestV2;
 
 /// <summary>
 /// TestMethod ducktype interface
 /// </summary>
-internal interface ITestMethod
+internal interface ITestMethod : IDuckType
```

Comment:
Maybe a silly question, but is this change needed?

Reply:
done in [0554177](https://github.com/DataDog/dd-trace-dotnet/pull/3341/commits/0554177875a4b6a3984eb3d833369811400eedba)